### ISSUE TITLE
Trakt Collection

### ISSUE THREAD

**Issue by Jeffers24** [umbrellaplug/umbrellaplug.github.io#258]
### Feature

When selecting TV Shows Collection, Umbrella is only showing the first X number of tv shows in my Trakt collection.
How do I get umbrella to show all the tv shows in my collection so that I can scroll up to the last show in my collection, rather than having to select a number of next pages showing X shows to get to the end of my collection?
![Screenshot (983)](https://user-images.githubusercontent.com/47518199/228158589-0a92591b-37d2-4afb-b3c3-a98a18ea2915.png)

**Comment by umbrellaplug** [comment]
Please check settings before opening issues. This is changed by changing the lost pagination options. They can be adjusted in settings under trakt and general.

**Comment by Jeffers24** [comment]
I checked through all the Umbrella settings before raising a functional request, not an issue.
Not being able to find a setting to change I didn't know if there was a settings option.
With your pointer to the Trakt options I found it and turned 'Paginate Lists when possible' to off, thank you.
![Screenshot (986)](https://user-images.githubusercontent.com/47518199/228263572-a0016015-932f-4869-bb14-2891dac39d35.png)
